SuperteamDAO/earn
# feat: edit sponsor profiles

## What does this PR do?

This PR resolves the issue mentioned in #477 

## How should this be manually tested?

- Go to the '/dashboard/listings' Page.
- At the top of the page, you will see an Edit Icon. Click on It.
- You will be redirected to the Edit Sponsor Profile Page ('/edit/sponsor')
- Edit your Sponsor Profile with the necessary data.
- Click on the Update Profile Button which will redirect you to the Dashboard Page.

## Existing Issue

- Dashboard Page (''/dashboard/listings'') doesn't update it's UI on the first-page render even though the store has been updated in the Edit Sponsor Profile Page. When you reload the page, it fetches the most recent data which is then updated in the UI
